app/src/main/java/com/keylesspalace/tusky/service/SendStatusService.kt

```diff
@@ -181,6 +182,25 @@ class SendStatusService : Service(), Injectable {
                 return@launch
             }
 
+            val isNew = statusToSend.statusId == null
+
+            if (isNew) {
+                media.forEach { mediaItem ->
+                    if (mediaItem.processed) {
+                        mastodonApi.updateMedia(mediaItem.id!!, mediaItem.description, mediaItem.focus?.toMastodonApiString())
+                            .fold({
+                            }, { throwable ->
+                                Toast.makeText(
+                                    applicationContext,
+                                    R.string.error_failed_set_caption,
+                                    Toast.LENGTH_SHORT
+                                ).show()
```

Comment:
Shouldn't it retry like it is done for all other failures? Or at least fail hard with a notification? Just showing a toast but continuing with posting the status is not what I would expect as a user

Reply:
Hm, it was like this before. (which of course might not have been optimal, but probably outside of the scope of this issue?)

Also I thought it was ok because alt text and focus point are not the main concerns on a status with an image.

But I can of course give retry + fail a look.

Reply:
Yes please have a look, maybe it is easy to do. If not, this is still better than before and we can improve it later.

Reply:
It uses now the same fail or retry logic as failures on the status send itself.